zebra/interface.c

```diff
@@ -1531,11 +1531,13 @@ static void interface_vrf_change(enum dplane_op_e op, ifindex_t ifindex,
 			if (exist_id != VRF_DEFAULT) {
 				vrf = vrf_lookup_by_id(exist_id);
 
-				flog_err(
-					EC_ZEBRA_VRF_MISCONFIGURED,
-					"VRF %s id %u table id overlaps existing vrf %s(%d), misconfiguration exiting",
-					name, ifindex, vrf->name, vrf->vrf_id);
-				exit(-1);
+				if (vrf) {
```

Comment:
Please add a error statement when the vrf lookup fails  here as well.

Reply:
Fixed.